bin/tessel-2.js

```diff
@@ -144,6 +144,15 @@ parser.command('provision')
   })
   .help('Authorize your computer to control the USB-connected Tessel');
 
+parser.command('restore')
+  .callback(callControllerCallback('restoreTessel'))
+  .option('openwrt', {
+    abbr: 'o',
+    flag: true,
+    help: 'Restore a factory version of OpenWrt'
+  })
+  .help('Restore your Tessel by installing the factory version of OpenWrt.');
```

Comment:
Why do we need the `openwrt` option on this command if there are no other options? From a user perspective, it looks like an unnecessary extra step.